web_widget_char_switchcase/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Web Char Switchcase Widget",
+    "version": "10.0.1.0.0",
+    "author": "Agile Business Group, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Web",
+    "website": "https://github.com/OCA/web/tree/"
```

Comment:
Thanks for you review! Since in https://github.com/OCA/maintainer-tools/blob/9766529146bcb61ce746623da74a23163199bdd3/template/module/__manifest__.py#L11 both formats are accepted, I'd prefer to keep the long format

Reply:
@pedrobaeza branch and module name are in the following line